salt/redmine/init.sls

```diff
@@ -0,0 +1,88 @@
+{% from 'lib.sls' import apache, create_user %}
+
+include:
+  - apache
+  - apache.modules.passenger
+  - mysql
+  - rvm
+
+{{ create_user(pillar.redmine.user) }}
+
+/home/{{ pillar.redmine.user }}/public_html:
+  file.directory:
+    - user: {{ pillar.redmine.user }}
+    - group: {{ pillar.redmine.user }}
+    - require:
+      - user: {{ pillar.redmine.user }}_user_exists
+
+redmine dependencies:
+  pkg.installed:
+    - pkgs:
+      - subversion
+      - ruby-dev
+      - libncurses5-dev
```

Comment:
Why is this package needed? https://redmine.org/projects/redmine/wiki/HowTo_Install_Redmine_50x_on_Ubuntu_2004_with_Apache2

Reply:
`ruby-dev` is required to install the Ruby Gems that the site runs on. 

I can't find where I sourced `libncurses5-dev` from, we had installed it on the current Redmine server so perhaps it used to be a requirement. 
I have removed the package and the application is working fine in testing.